api/src/services/users/users.resolver.ts

```diff
@@ -59,10 +59,10 @@ export const usersPatchResolver = resolve<UsersPatch, HookContext>({
      * Because we don't take in consideration the user password at the creation.
      * It will be defined by the user himself after the signup verification.
      */
-    password: async (_password, data, context) => {
-      const p = generatePassword(context.app.get('settings').passwordPolicy)
-      return await passwordHash({ strategy: 'local' })(p, data, context)
-    },
+    // password: async (_password, data, context) => {
```

Comment:
Why do you keep it if no longer required

Reply:
You're right !
Done